packages/ui-extensions/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './types';
```

Comment:
I think this is okay because we're importing explicitly as a type in argo-admin - https://github.com/Shopify/argo/pull/131/files#diff-d9f7c7d2fb6a1b6c7fe5317ad99f1a16380074c311023c3edf920ba60667b575R2 

Let's wait for @iainmcampbell to comment to be sure though.

Reply:
I tried explicitly exporting the types from this package, didn't work

Reply:
The script will not currently parse types imported from another package

Reply:
This might be a good forcing function to get that working though 😄

Reply:
😅

Reply:
Thanks @iainmcampbell